tools/sha256_generator.sh

```diff
@@ -20,7 +20,7 @@ set -o nounset
 set -o pipefail
 
 cd $(mktemp -d)
-KUBE_VERSION="${KUBE_VERSION:-v1.24.2}"
+KUBE_VERSION="${KUBE_VERSION:-v1.28.0}"
```

Comment:
FYI we don't need this, as we can set `KUBE_VERSION` env before calling the script

Reply:
Done.